### DISCUSSION TITLE
Netbox 3.3.2 export template issue, help

### DISCUSSION THREAD

**Discussion by settlej** [netbox-community/netbox#11580] [Q&A] [answered]
I'm unable to export "manufacturer" using the custom export template via jinja2.

{% for device in queryset %} 
  {{device.name}}
  {{device.manufacturer}}
{% endfor %}

Every other field works, even custom fields.  Does anyone else see this problem?

**Comment by mtinberg** [comment, marked as answer]
I don't think manufacturer is a direct reference from Device, it's joined through Device Type, and even if it were it'd have it's own key/value pairs so what you probably want is {{device.device_type.manufacturer.name}} or something like that.  You could also print another field like id, display, slug, and so on.

Taking a look at the /api/dcim/devices/ output can help you get a better sense of what format the returned data actually looks like


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: settlej ***@***.***>
Sent: Wednesday, January 25, 2023 9:24 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] Netbox 3.3.2 export template issue, help (Discussion #11580)


I'm unable to export "manufacturer" using the custom export template via jinja2.

{% for device in queryset %}
{{device.name}}
{{device.manufacturer}}
{% endfor %}

Every other field works, even custom fields. Does anyone else see this problem?

**Reply by settlej** [reply]
Thank you!
